docs/traits.adoc

```diff
@@ -62,6 +62,22 @@ The following is a list of common traits that can be configured by the end users
   +
   It's enabled by default.
 
+| deployer
+| Kubernetes, OpenShift
+| Configure deployer behavior.
+  +
+  +
+  It's enabled by default.
+
+
+[cols="m,"]
+!===
+
+! deployer.container-image
```

Comment:
Is this a "third" way of doing things?

Reply:
it is the only way now and it is used by both the deployment and the knative_service traits